javascript-practice/src/styles/utils/variables.css

```diff
@@ -27,6 +27,7 @@
   --boder-radius-input-button: 8px;
   --boder-radius-login-input: 4px;
   --boder-radius-toast: 5px;
+  --boder-radius-box: 6px;
```

Comment:
--boder-radius-box: 6px; => BORDER, btw boder-radius-box does not make sense, pls choose another name

Reply:
You can set border directly. Why use it as variable? Use variable for reuse purpose